src/content/smart-bulletins/036.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Smart Bulletin No. 036"
-intro: "Use of GSA SmartPay® Charge Cards to Pay for Electric Vehicle Charging Pursuant to the “Charging Helps Agencies Realize General Efficiencies Act” or “CHARGE Act”).."
+title: "Use of GSA SmartPay® Charge Cards to Pay for Electric Vehicle Charging Pursuant to the “Charging Helps Agencies Realize General Efficiencies Act” or “CHARGE Act”).."
```

Comment:
Please remove the ").." per meeting with John B.

Reply:
updated